teamhero/hawc
# Appdev 3864 allow sorting on animal endpoints list by noel and loel

To test this, please to the following:

1) Start django server once virtual environment is activated and in project directory for that terminal:
`python manage.py runserver 0:8000`

2) Start NPM server once virtual environment is activated and in project directory for that terminal:
`npm start`

3) Open browser to this address:
localhost:8000/ani/assessment/483/endpoints/

4) If you need to login, login to see this page.

5) Below the "Assessment endpoints" title, click "Filter endpoints" to see the sub-form.

6) Select "NOEL" in the Order by dropdown list and click the "Apply filters" button.

7) Once executed, you will see the integer numbers that are not negative show up first and the values in the NOEL column that are "-" dashes at the end of the results.

8) Do steps 6 and 7 again but with "LOEL" selected in the Order by dropdown list to see results.
